.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
     "files.associations": {
-        "*.csproj": "msbuild",
+        "*.csproj": "xml",
```

Comment:
what is this for?

Reply:
vscode was not giving me syntax highlighting/formatting for these, will revert prior to undrafting.

Reply:
I think you need an MSBuild extension that has "msbuild" language syntax highlighting plus more (like intellisense)

Reply:
Thanks, it looks like https://marketplace.visualstudio.com/items?itemName=tintoy.msbuild-project-tools is what I was looking for, which is also included in extensions.json in this repo.